CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,50 @@
+# Contributor Covenant Code of Conduct
```

Comment:
can you move this to `.github/`?

Reply:
yes, i've removed this and updated the Readme to refer to the existing file that's in the .github/ directory